timers.js

```diff
@@ -1,4 +1,34 @@
 
+var start = $("#start");
+var reset = $("#reset");
+var pause = $("#pause");
+var seconds = 0;
+var timerID;
 
+function countUp() {
+    seconds = seconds+.01;
+    $("h1").text("Time elapsed: "+seconds.toFixed(2));
+}
 
+function startTime() {
+    if (timerID == null) {
+    $("h1").text("Time elapsed: "+seconds.toFixed(2));
+    timerID = setInterval(countUp, 10);
```

Comment:
:+1: funny how this short of a `setInterval` makes the timer shake pretty intensely.